test/perf-flights.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const autocannon = require('autocannon')
+
+const url = 'http://localhost:9080/rest/api/flights'
+const postData = 'fromAirport=BOM&toAirport=AMS&fromDate=Thu%20Jun%2021%202018%2000%3A00%3A00%20GMT-0500%20(Central%20Daylight%20Time)&returnDate=Thu%20Jun%2021%202018%2000%3A00%3A00%20GMT-0500%20(Central%20Daylight%20Time)&oneWay=false'
+
+autocannon({
+  title: 'acmeair-flights',
+  url,
+  method: 'POST',
+  body: postData,
+  headers: {
+    'Content-Type': 'application/x-www-form-urlencoded'
+  },
+  connections: 10,
+  pipelining: 1,
+  duration: 30
+}, console.log)
```

Comment:
V8 optimizes code as it runs, so it's better to run the test twice, and only print the second result. The results should improve.

Also, you should add https://github.com/mcollina/autocannon#autocannontrackinstance-opts so that it prints a nice progress bar (and output) as it runs.